listener.js

```diff
@@ -85,14 +85,20 @@ server.post("/", (payload, res) => {
                             ipaddr = await cli_exec("grep -A1 \"" + device.name.replace('+', '.*') + "\" /var/db/dhcpd_leases", 'device_ipaddr');
                         }
                     }
-                    let reopen = await cli_exec("curl http://" + ipaddr + ":8080/restart", "device_command");
+                    let reopen = await cli_exec("curl --connect-timeout 10 -m 10 http://" + ipaddr + ":8080/restart", "device_command");
```

Comment:
```suggestion
                    const reopen = await cli_exec("curl --connect-timeout 10 -m 10 http://" + ipaddr + ":8080/restart", "device_command");
```
Since reopen value is never changed